docs/user_guide/api/targets.md

```diff
@@ -193,4 +193,43 @@ Returns an empty body if successful.
             "Error Text"
         ]
     }
-    ```
\ No newline at end of file
+    ```
+    
+## `PATCH /api/v1/targets/{id}/subscriptions`
+
+Updates existing subscriptions for the target ID
+    
+Returns an empty body if successful.
+
+=== "Request"
+    ```bash
+    curl --request PATCH gnmic-api-address:port/api/v1/targets/192.168.1.131:57400/subscriptions -d '{"subscriptions": ["sub1", "sub2"]}'
+    ```
+=== "200 OK"
+    ```json
+    ```
+=== "404 Not found"
+    ```json
+    {
+        "errors": [
+            "target $target not found"
+        ]
+    }
+    ```
+=== "400 Bad Request"
+    ```json
+    {
+        "errors": [
+            "subscriptions not found",
+            "subscription $subscription does not exist"
+        ]
```

Comment:
Looking at `UpdateTargetSubscription`, I don't see 2 errors being returned at the same time. I think these are 2 distinct responses.

Reply:
right.. i wasn't sure if this was trying to capture all potential 400 responses or just serve as an example of a single bad response, so i opted for all 🤷

Reply:
i can adjust the example to return a single response

Reply:
I think it makes sense to adjust it if you don't mind.

Reply:
of course dude!

Reply:
i _think_ that solves it.. if you want i can work on adding some real tests to the api side

Reply:
nice, tests would be great.